create_infotexts.py

```diff
@@ -471,15 +513,14 @@ def generate_list_of_stripped_keywords(self, keyword_string):
         :return: list of keywords.
         """
         keywords_list = keyword_string.split(", ")
-        keywords_list_low = [kw.lower() for kw in keywords_list]
```

Comment:
Why the change? Wasn't the reason for converting these that the keywords in the metadata were a mixture of upper and lower case?